sample-input/gradients/one-directional/one-directional-cmfd.py

```diff
@@ -0,0 +1,83 @@
+import numpy
+from openmoc import *
+import openmoc.log as log
+import openmoc.plotter as plotter
+from openmoc.options import Options
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+options = Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+log.set_log_level('NORMAL')
+log.py_printf('TITLE', 'Simulating a one group homogeneous, one directional'
+    ' gradient...')
+
+###############################################################################
+###########################   Creating CMFD Mesh    ###########################
+###############################################################################
+log.py_printf('NORMAL', 'Creating Cmfd mesh...')
+
+cmfd = Cmfd()
+cmfd.setMOCRelaxationFactor(0.6)
+cmfd.setSORRelaxationFactor(1.5)
+cmfd.setLatticeStructure(51,1)
+cmfd.setGroupStructure([1])
```

Comment:
I would recommend getting rid of this line as CMFD is using the same group structure as MOC. I'm actually surprised this even works since the group structure should start at 1 and end with the number of MOC energy groups + 1.